### PR TITLE
Update Rule “keep-track-of-a-parking-lot-for-topics/rule”

### PR DESCRIPTION
**Pull request by UlyssesMaclaren**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/keep-track-of-a-parking-lot-for-topics/rule.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 
-When you identify topics that do need to be hashed out, but are off topic and don’t need to be covered in this meeting, keep track of them in a “Parking Lot”.  
-E.g. During the meeting you can call out _“Off topic. I think that topic should be saved for the “Parking Lot”. Who is interested in joining that?”_
+When you identify topics that do need to be hashed out, but are off topic and don’t need to be covered in this meeting, keep track of them in a “Parking Lot”.
+
+E.g. During the meeting you can call out *“Off topic. I think that topic should be saved for the “Parking Lot”. Who is interested in joining that?”*
 
 <!--endintro-->
 
+![Figure: Video - See the Parking Lot in action: www.youtube.com/watch?v=-UUrLxNBK_g](parking-lot.jpg)
```
Comment (bradystroud) [inline rules/keep-track-of-a-parking-lot-for-topics/rule.md:23]: Embed the YouTube video. See www.ssw.com.au/rules/rule (about halfway down the page)
```suggestion
`youtube: www.youtube.com/watch?v=-UUrLxNBK_g`
**Figure : See the parking lot in action**
```

**Review by bradystroud — CHANGES_REQUESTED** [review]
Would be much better if the video was embedded. See my suggestion

**Comment by UlyssesMaclaren** [comment]
Adam requested it this way so we could specify the Parking Lot in the background of the image

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by bradystroud** [comment]
> Adam requested it this way so we could specify the Parking Lot in the background of the image

Understood. No problem

**Review by bradystroud — APPROVED** [review]

**Pull request merged**